eis_qgis_plugin/eis_processing/algorithms/exploratory_analysis/feature_importance.py

```diff
@@ -0,0 +1,69 @@
+from qgis.core import (
+    QgsProcessing,
+    QgsProcessingParameterFile,
+    QgsProcessingParameterMultipleLayers,
+    QgsProcessingParameterNumber,
+    QgsProcessingParameterRasterLayer,
+)
+
+from eis_qgis_plugin.eis_processing.eis_processing_algorithm import EISProcessingAlgorithm
+
+
+class EISFeatureImportance(EISProcessingAlgorithm):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self._name = "feature_importance"
+        self._display_name = "Feature importance"
+        self._group = "Exploratory analysis"
+        self._group_id = "exploratory_analysis"
+        self._short_help_string = """
+        Evaluate the feature importance of a Sklearn classifier or regressor.
+        """
+
+    def initAlgorithm(self, config=None):
+        self.alg_parameters = [
+            "model_file",
+            "input_rasters",
+            "target_labels",
+            "n_repeats",
+            "random_state",
+        ]
+
+        model_file_param = QgsProcessingParameterFile(
+            name=self.alg_parameters[0], description="Model file", fileFilter='.joblib (*.joblib)'
+        )
+        model_file_param.setHelp("The model file.")
+        self.addParameter(model_file_param)
+
+        evidence_data_param = QgsProcessingParameterMultipleLayers(
+            name=self.alg_parameters[1], description="Feature data", layerType=QgsProcessing.TypeRaster
+        )
+        evidence_data_param.setHelp(
+            "Data on which feature importance will be computed."
+        )
+        self.addParameter(evidence_data_param)
```

Comment:
Could mention here that the data and order of rasters need to match the data used in training.